lib/client/mfa.go

```diff
@@ -42,7 +42,7 @@ func PromptMFAChallenge(ctx context.Context, proxyAddr string, c *proto.MFAAuthe
 		return &proto.MFAAuthenticateResponse{}, nil
 	// TOTP only.
 	case c.TOTP != nil && len(c.U2F) == 0:
-		totpCode, err := prompt.Input(os.Stdout, os.Stdin, fmt.Sprintf("Enter an OTP code from a %sdevice", promptDevicePrefix))
+		totpCode, err := prompt.Input(os.Stderr, os.Stdin, fmt.Sprintf("Enter an OTP code from a %sdevice", promptDevicePrefix))
```

Comment:
Just curious why this change to stderr?

Reply:
Because of `kubectl`'s exec plugins.
We configure `kubectl` to exec a `tsh` subcommand every time it needs to get credentials to connect.
`kubectl` expects to read the credentials from `stdout` of the execed process.
Anything written to `stderr` will be forwarded to `kubectl`'s own `stderr` to allow interactive commands like this.